packages/breadcrumb/src/BreadcrumbContainer.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+
+import useBreadcrumb, { IUseBreadcrumbReturnValue } from './useBreadcrumb';
+
+export interface IBreadcrumbContainerProps {
+  render?: (options: IUseBreadcrumbReturnValue) => React.ReactElement;
+  children?: (options: IUseBreadcrumbReturnValue) => React.ReactElement;
+}
+
+const BreadcrumbContainer: React.FunctionComponent<IBreadcrumbContainerProps> = ({
+  children,
+  render = children
+}) => {
+  return render!(useBreadcrumb());
+};
+
+BreadcrumbContainer.propTypes = {
```

Comment:
On the `react-components` side, we've been removing `PropTypes` – in which case, the dependency would also be removed from package.json

Reply:
For all the components that are external facing we should include `propTypes` as well as the interfaces. Not all consumers are using TS so this would help them.